<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AzureKinectClass</class>
 <widget class="QMainWindow" name="AzureKinectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1304</width>
    <height>832</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AzureKinect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="Ak::KinectWidget" name="openGLWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>512</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelPID">
          <property name="text">
           <string>Pid:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDepth_Image"/>
    <addaction name="actionColour_Image"/>
    <addaction name="actionIR_Image"/>
    <addaction name="actionBody_Shadow"/>
    <addaction name="actionBody_Skeleton"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
    <addaction name="actionDepth_Image_2"/>
    <addaction name="actionColour_Image_2"/>
    <addaction name="actionIR_Image_2"/>
    <addaction name="actionBody_Skeleton_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuRecord"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDepth_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Depth Image</string>
   </property>
  </action>
  <action name="actionColour_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colour Image</string>
   </property>
  </action>
  <action name="actionBody_Shadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Body Shadow</string>
   </property>
  </action>
  <action name="actionBody_Skeleton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Body Skeleton</string>
   </property>
  </action>
  <action name="actionIR_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IR Image</string>
   </property>
  </action>
  <action name="actionDepth_Image_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Depth Image</string>
   </property>
  </action>
  <action name="actionColour_Image_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Colour Image</string>
   </property>
  </action>
  <action name="actionIR_Image_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IR Image</string>
   </property>
  </action>
  <action name="actionBody_Skeleton_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Body Skeleton</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ak::KinectWidget</class>
   <extends>QWidget</extends>
   <header>kinectwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="AzureKinect.qrc"/>
 </resources>
 <connections/>
</ui>
